apiv6/kentikapi/httputil/httputil_test.go

```diff
@@ -106,6 +107,48 @@ func TestRetryingClientWithSpyHTTPTransport_Do(t *testing.T) {
 	}
 }
 
+func TestRetryAfter(t *testing.T) {
```

Comment:
```suggestion
func TestRetryingClient_Do_HandlesRetryAfterHeader(t *testing.T) {
```
To keep test naming consistent and to make the name informative.

Reply:
I still think that `TestRetryingClient` is better prefix that `TestNewRetryingClient`, because we test functionality of retrying client, not only its constructor.

Reply:
Done